<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="WidgetMainHeader" name="headerWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="XYNavListView" name="navbarWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="WidgetMainTabbed" name="tabbedWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="WidgetMainStatus" name="statusWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetMainHeader</class>
   <extends>QWidget</extends>
   <header location="global">WidgetMainHeader.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetMainTabbed</class>
   <extends>QWidget</extends>
   <header location="global">WidgetMainTabbed.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetMainStatus</class>
   <extends>QWidget</extends>
   <header location="global">WidgetMainStatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYNavListView</class>
   <extends>QWidget</extends>
   <header location="global">XYNavListView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
